src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+                    ._accordionActiveIndex === 0
+                    ? '--open'
+                    : ''}"
+                >
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-0"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-1"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-2"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-3"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-4"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-5"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-6"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-0-item-7"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                </ul>
+              </div>
+
+              <div class="o-footer__main">
+                <slot
+                  name="column-1-title-desktop"
+                  class="o-footer__title-desktop"
+                ></slot>
+                <button
+                  class="o-footer__accordion-button"
+                  @click="${() => this._handleAccordionClick(1)}"
+                >
+                  <slot name="column-1-title" class="o-footer__title"></slot>
+                  <span
+                    class="o-footer__accordion-button-caret o-footer__accordion-button-caret${this
+                      ._accordionActiveIndex === 1
+                      ? '--open'
+                      : ''}"
+                  >
+                    ${svg([CaretSvg || ''])}
```

Comment:
ditto for extracting expression, which - since repeated here - now also violates DRY